examples/cifar10_cnn_capsule.py

```diff
@@ -14,14 +14,18 @@
 
 from __future__ import print_function
 from keras import backend as K
-from keras.engine.topology import Layer
+from keras.layers import Layer
 from keras import activations
 from keras import utils
 from keras.datasets import cifar10
 from keras.models import Model
 from keras.layers import *
 from keras.preprocessing.image import ImageDataGenerator
 
+if K.backend() == 'mxnet':
```

Comment:
thanks for this catch, I have updated the[ issue list](https://github.com/awslabs/keras-apache-mxnet/issues/79)